wolfcrypt/src/rsa.c

```diff
@@ -4834,7 +4834,9 @@ int wc_MakeRsaKey(RsaKey* key, int size, long e, WC_RNG* rng)
             err = MEMORY_E;
     }
 
-    SAVE_VECTOR_REGISTERS(err = _svr_ret;);
+    if (err == MP_OKAY) {
```

Comment:
In most cases this macro is a do nothing `#define SAVE_VECTOR_REGISTERS(...) WC_DO_NOTHING` and `#define WC_DO_NOTHING do {} while (0)`. I hope no static analyzers will flag this as dead code. Please confirm there are no other uses of `SAVE_VECTOR_REGISTERS` that have this issue. Please use @douzzer for review.